llvm/test/CodeGen/X86/apx/inline-asm-r-constraint.ll

```diff
@@ -0,0 +1,16 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: not llc -mtriple=x86_64 < %s 2>&1 | FileCheck %s --check-prefix=ERR
+; RUN: not llc -mtriple=x86_64 -mattr=+egpr < %s 2>&1 | FileCheck %s --check-prefix=ERR
+; RUN: llc -mtriple=x86_64 -mattr=+egpr,+inline-asm-use-gpr32 < %s | FileCheck %s
```

Comment:
I find you also extend the `l` constraint. Should we have a test for it? And how about `q` https://github.com/llvm/llvm-project/pull/73529 ?

I think it should be extended too.

Reply:
agree, extended `q` in [bf3a53c](https://github.com/llvm/llvm-project/pull/92338/commits/bf3a53c0b7fc1828572f771e4772d25062110dc0)